### ISSUE TITLE
Chamfer rail motor plate has too short slots

### ISSUE THREAD

**Issue by Jaknil** [fellesverkstedet/fabricatable-machines#9]
The current chamfer rail motor plate has too short slots for the pinion to be moved completely out of the rack teeth when tuning the glide blocks. They need to be extended at least 3.6 mm on the inside for it to be possible. See pic.

![motor plate](https://user-images.githubusercontent.com/3476653/33849296-8783b620-deb0-11e7-8485-1b02a9e4e808.PNG)

**Comment by JensDyvik** [comment]
Thank you for good feedback and well explaining screenshot. The missing feature has been added to the chamfer rail generator and the example files have been updated here https://github.com/fellesverkstedet/fabricatable-machines/tree/master/chamferrail

The downside of the is that the adjustment holes become very long. Especially since it is nice to to have the glidblock symmetric for ease off assembly. I increased the default wall thickness outside the holes from 4mm to 6mm for keeping a robust motor plate, also in weak materials
